public/css/style.css

```diff
@@ -468,8 +473,12 @@ tr:last-child td {
   height: 56px;
   line-height: 56px;
   padding-left: 20px;
-  display: none;
   pointer-events: none;
+  transform: translateY(56px);
+  opacity: 0;
+  transition-property: transform,opacity;
```

Comment:
Does the short notation work with multiple properties? (transform, opacity). That's the reason I used the long notation here. Looks like an alternative would be something like: 
`transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out;`, but I'm not sure it's better, as we end up repeating ourselves more.